### PR TITLE
[ticket/14596] Deny the installation of Ascraeus under PHP7

### PR DESCRIPTION
**Pull request by 3D-I**
PHPBB3-14596

### CODE REVIEW

**Comment by Mauron** [comment]
You should name the branch after the ticket name, e.g. _ticket/14596_.

**Comment by DavidIQ** [comment]
Branch name is not the problem and it doesn't really matter what the branch name is.  Something is wrong with the commit messages.

**Comment by 3D-I** [comment]
Yes David, correct. :)

This time I am getting that the  Travis build fails just under PHP 5.3.3 (as usual), but this time with the following error:

> Unexpected EOF encountered 

Aborting commit. 

Inspecting commit message of commit 30c822466c24e7952d8e36e1d223ac187bc67524

Unexpected EOF encountered 

Aborting commit. 

The command "sh -c "if [ '$TRAVIS_PHP_VERSION' = '5.3.3' -a '$DB' = 'mysqli' -a '$TRAVIS_PULL_REQUEST' != 'false' ]; then git-tools/commit-msg-hook-range.sh origin/$TRAVIS_BRANCH..FETCH_HEAD; fi"" exited with 1.

/me gonna read that article though.

**Comment by 3D-I** [comment]
Also, I don't see where I am wrong (the wiki could be wrong here)
have a look at this commit by Senky

https://github.com/phpbb/phpbb/pull/4223

**Comment by DavidIQ** [comment]
It's also failing (for the same reaso as this one) and hasn't been merged.  Your commit message does not follow the guideline.  Please take a look at it again.

**Comment by Skouat** [comment]
Maybe you should add an empty line between the title and the ticket number of each commit.

**Comment by 3D-I** [comment]
OK David I will.

It is my first commit to phpBB so I need to be lead by hand, a clear and visive example will do.

TIA

**Comment by 3D-I** [comment]
@Skouat  I believe I did.. into the comment post area I put a blank new line and then the ticket number

> blank line + CR

> PHPBB3-14596

**Comment by 3D-I** [comment]
@DavidIQ 
I did something in my previous ticket https://github.com/phpbb/phpbb/pull/4293
but there I gor a different error, alsways under 5.3.3

There is something I can't understand but should be ovious, as usual...

File: phpBB/language/en/install.php [hunk 1]
```diff
@@ -300,11 +300,11 @@
 	'PHP_REGISTER_GLOBALS'			=> 'PHP setting <var>register_globals</var> is disabled',
 	'PHP_REGISTER_GLOBALS_EXPLAIN'	=> 'phpBB will still run if this setting is enabled, but if possible, it is recommended that register_globals is disabled on your PHP install for security reasons.',
 	'PHP_SAFE_MODE'					=> 'Safe mode',
-	'PHP_SETTINGS'					=> 'PHP version and settings',
-	'PHP_SETTINGS_EXPLAIN'			=> '<strong>Required</strong> - You must be running at least version 5.3.3 of PHP in order to install phpBB. If <var>safe mode</var> is displayed below your PHP installation is running in that mode. This will impose limitations on remote administration and similar features.',
+	'PHP_SETTINGS'					=> 'PHP versions range and settings',
```
Comment (VSEphpbb) [inline phpBB/language/en/install.php]: This text makes no sense grammatically.

**Comment by 3D-I** [comment]
@VSEphpbb  .. 
yup, I will change it, I am going to create a new PR after having read the Wiki a little more. Thx.

**Comment by 3D-I** [comment]
Closing for the incoming new PR's sake.

**Pull request closed without merging**